### PR TITLE
Rails 5.1 test

### PR DESCRIPTION
**Pull request by aaackerman**

### CODE REVIEW

**Review by dglatstein — APPROVED** [review]

File: app/controllers/s3_multipart/uploads_controller.rb [hunk 1]
```diff
@@ -28,50 +28,53 @@ def update
...
-        rescue => e
-          logger.error "EXC: #{e.message}"
-          response = {
-            error_message: t("s3_multipart.errors.update"),
-            error: e.message
-          }
-        ensure
-          render :json => response
-        end
+    def sign_batch
```
Comment (jackpope) [inline app/controllers/s3_multipart/uploads_controller.rb]: I think the private methods should remain indented to match the style of the rest of this repo

File: app/controllers/s3_multipart/uploads_controller.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@ class UploadsController < ApplicationController
 
     def create
       begin
-        upload = Upload.create(params)
+        upload = Upload.create(upload_params)
```
Comment (jackpope) [inline app/controllers/s3_multipart/uploads_controller.rb]: Might be more readable to do the casting here?

**Review by jackpope — APPROVED** [review]

File: app/controllers/s3_multipart/uploads_controller.rb (hunk 1 above)
Comment (aaackerman) [reply app/controllers/s3_multipart/uploads_controller.rb]: 👎 but ok! haha

File: app/controllers/s3_multipart/uploads_controller.rb (hunk 2 above)
Comment (aaackerman) [reply app/controllers/s3_multipart/uploads_controller.rb]: Yeah I originally did this but had to update everywhere. Maybe i could name it better?

Comment (jackpope) [reply app/controllers/s3_multipart/uploads_controller.rb]: Oh yeah didn't even see it in the other methods. NBD then

**Pull request merged**